server/src/modes/template/modifierProvider.ts

```diff
@@ -0,0 +1,98 @@
+// tslint:disable: max-line-length
+import { MarkupContent } from 'vscode-languageserver-types';
+
+export interface Modifier {
+  label: string;
+  documentation?: string | MarkupContent;
+}
+
+function genModifier(label: string, documentation?: string | MarkupContent) {
+  return { label, documentation };
+}
+
+const eventModifiers = [
+  genModifier('stop', 'The event propagation will be stopped.'),
+  genModifier('prevent', 'The event will no longer default action.'),
+  genModifier('capture', 'Use capture mode when adding the event listener.'),
+  genModifier('self', 'Only trigger handler if event.target is the element itself.'),
+  genModifier('once', 'The event will be triggered at most once.'),
+  genModifier(
+    'passive',
+    "The scroll event's default behavior (scrolling) will happen immediately, instead of waiting for `onScroll` to complete."
```

Comment:
I think this is too specific. Something better would be:

```suggestion
    "Indicates that the event listener will never call `preventDefault`. Same as `addEventListener`'s `passive` option."
```

You might add the `scroll.passive` as a further explanation below, such as:

```
For example, `scroll.passive` allows the scroll event's default behavior (scrolling) to happen
immediately, instead of waiting for `onScroll` to complete in case it contains `preventDefault` calls.
```